### PR TITLE
LPS-119446

### PR DESCRIPTION
**Pull request by hhuijser**

### CODE REVIEW

**Comment by hhuijser** [comment]
ci:reopen

**Comment by hhuijser** [comment]
@brianchandotcom unrelated failures only: https://github.com/hhuijser/liferay-portal/pull/5571

I hate for these pulls to get so big, but splitting em up (with unrelated failures in all of em) just slows things down like crazy, since all these pulls are depending on each other as well.

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/9ddda30226...6f5e43149c

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java [hunk 1]
```diff
@@ -224,7 +224,6 @@ public CommerceOrder addCommerceOrder(
 		commerceOrder.setCompanyId(user.getCompanyId());
 		commerceOrder.setUserId(userId);
 		commerceOrder.setUserName(user.getFullName());
-		commerceOrder.setCreateDate(now);
```
Comment (arboliveira) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java:227]: @hhuijser 

```
[java] 1: For better performance, the variable declaration for 'now' should be moved inside the else-statement on line '257': ./modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderLocalServiceImpl.java 221 (Checkstyle:VariableDeclarationAsUsedCheck)
```

File: modules/apps/portal-lock/portal-lock-service/src/main/java/com/liferay/portal/lock/service/impl/LockLocalServiceImpl.java [hunk 2]
```diff
@@ -221,8 +221,6 @@ else if (!renew) {
 
 		Date now = new Date();
 
-		lock.setCreateDate(now);
```
Comment (arboliveira) [inline modules/apps/portal-lock/portal-lock-service/src/main/java/com/liferay/portal/lock/service/impl/LockLocalServiceImpl.java:224]: @hhuijser 

```
[java] 2: For better performance, the variable declaration for 'now' should be moved inside the else-statement on line '227': ./modules/apps/portal-lock/portal-lock-service/src/main/java/com/liferay/portal/lock/service/impl/LockLocalServiceImpl.java 222 (Checkstyle:VariableDeclarationAsUsedCheck)
```

**Pull request closed without merging**